docs/getting-started-guides/docker.md

```diff
@@ -14,7 +14,8 @@ Here's a diagram of what the final result will look like:
 
 1. You need to have docker installed on one machine.
 2. Decide what Kubernetes version to use.  Set the `${K8S_VERSION}` variable to
-   a released version of Kubernetes >= "1.2.0-alpha.7"
+   a released version of Kubernetes >= "1.2.0-alpha.7". Get all available versions by
+   ```curl -k -s -X GET https://gcr.io/v2/google_containers/hyperkube-amd64/tags/list | jq -r '.tags[]'```
```

Comment:
Most systems don't have jq installed; can you do this in regex with grep -E?

Also, have a blanks space on line 18, have line 19 read "```shell", have line 20 be the command, and line 21 be "````"